public/js/core.js

```diff
@@ -366,6 +380,10 @@ function fromEther (value) {
     return web3.utils.fromWei(BigInt(value).toString(), 'ether')
 }
 
+function convertClaimTax(value) {
+    return value*0.15/2767011611056432742
```

Comment:
where does 2767011611056432742 come from? can you send a ref here? thanks

Reply:
Came from cryptoblades contract [(here)](https://bscscan.com/address/0x9c29136bc1617b011e6c125b616fbaba1d311b6e#code).

Inside "File 4"'s contract:
![image](https://user-images.githubusercontent.com/13718516/127758628-54dde173-67b6-40a1-825e-8a819ff691dd.png)

```
int128 public constant REWARDS_CLAIM_TAX_MAX = 2767011611056432742; // = ~0.15 = ~15%
```

Reply:
Also just noticed "REWARDS_CLAIM_TAX_MAX" function to retrieve the maximum claim tax value (2767011611056432742).

Just commited another version getting this value on document ready.

Reply:
so that's the max tax? what about the daily -1%? where did you apply it?

Reply:
or the contract itself subtract it automatically?

Reply:
Exactly,"getOwnRewardsClaimTax" function returns the updated value, that can be compared against the max value in order to retrieve current tax value in %.

Reply:
i see